pkg/cmd/ui/ui.go

```diff
@@ -184,12 +184,13 @@ func (o UIOptions) waitForForwarding(localURL string) error {
 	return o.RetryUntilTrueOrTimeout(time.Minute, time.Second*3, func() (b bool, e error) {
 		log.Logger().Debugf("Checking the status of %s", localURL)
 		resp, err := http.Get(localURL) // #nosec
-		if err != nil || (resp != nil && resp.StatusCode != 200) {
+		respSuccess := resp != nil && (resp.StatusCode == 200 || resp.StatusCode == 401)
+		if err != nil || !respSuccess {
 			log.Logger().Debugf("Returned err: %+v", err)
```

Comment:
If err is nil and response success is false, then the logger will not give any results which can be used for debugging, may be include the value of response success ... or just break it into two separate checks, one for error and one for respSuccess? thoughts?

Reply:
true in this case it doesn't tell the user much. However it is already merged ;). On the other hand none of the errors would clarify to the user what the problem is other than not reaching JXUI which is obvious.